AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/AgentsRoutes.java

```diff
@@ -87,4 +74,11 @@ public Object deleteAgent(Request req, Response res, int id) throws Exception {
 
         return new Object();
     }
+
+    public void verifyPermission(Request req) throws FailedRequestException {

```

Comment:
Should perhaps include 'admin' in there somewhere - otherwise it's not clear what role is being tested.